code/modules/mob/living/carbon/human/human_damage.dm

```diff
@@ -26,9 +26,8 @@
 				if(use_brain_mod)
 					amount *= dna.species.brain_mod
 			sponge.damage = clamp(sponge.damage + amount, 0, 120)
-			if(sponge.damage >= 120)
+			if(sponge.damage >= 120 && death())
 				visible_message("<span class='alert'><B>[src]</B> goes limp, [p_their()] facial expression utterly blank.</span>")
```

Comment:
I'm not convinced we need this message to be honest because it is quite similar to the default death message. Would it be better rewritten or just removed?

Reply:
I agree, doctors/robos don't get any information out of it that they can't get from a brief inspection and it's not really relevant to anyone else.

Reply:
I've never even seen this message before, I'd be fine with entirely removing it